src/channel_internal.c

```diff
@@ -222,27 +221,16 @@ static void dispose_channel_op(void* context)
     (void)context;
 }
 
-static int enqueue_operation(
-    THANDLE(CHANNEL_INTERNAL) channel_internal,
-    THANDLE(ASYNC_OP)* out_op,
-    THANDLE(RC_STRING) pull_correlation_id,
-    PULL_CALLBACK pull_callback,
-    void* pull_context,
-    THANDLE(RC_STRING) push_correlation_id,
-    PUSH_CALLBACK push_callback,
-    void* push_context,
-    THANDLE(RC_PTR) data
-)
+static int enqueue_operation(THANDLE(CHANNEL_INTERNAL) channel_internal, THANDLE(ASYNC_OP)* out_op, THANDLE(RC_STRING) pull_correlation_id, PULL_CALLBACK pull_callback, void* pull_context, THANDLE(RC_STRING) push_correlation_id, PUSH_CALLBACK push_callback, void* push_context, THANDLE(RC_PTR) data)
 {
     int result;
 
-
     /*Codes_SRS_CHANNEL_INTERNAL_43_103: [ channel_internal_pull shall create a THANDLE(ASYNC_OP) by calling async_op_create with cancel_op as cancel. ]*/
     /*Codes_SRS_CHANNEL_INTERNAL_43_119: [ channel_internal_push shall create a THANDLE(ASYNC_OP) by calling async_op_create with cancel_op as cancel. ]*/
     THANDLE(ASYNC_OP) async_op = async_op_create(cancel_op, sizeof(CHANNEL_OP), alignof(CHANNEL_OP), dispose_channel_op);
     if (async_op == NULL)
     {
-        LogError("Failure in async_op_create(cancel_op, sizeof(CHANNEL_OP), alignof(CHANNEL_OP), dispose_channel_op)");
+        LogError("Failure in async_op_create(cancel_op = %p, sizeof(CHANNEL_OP) = %" PRIu32 ", alignof(CHANNEL_OP) = %" PRIu32 ", dispose_channel_op = %p)", cancel_op, (uint32_t)sizeof(CHANNEL_OP), (uint32_t)alignof(CHANNEL_OP), dispose_channel_op);
```

Comment:
>(uint32_t) [](http://example.com/codeflow?start=168&length=10)

Why do we need this cast? size_t can be printed with %zu. And that is usually what we do.

Reply:
Here and everywhere else.

Reply:
Fixed. I put the cast there because that is the type that `async_op_create` accepts.